src/test/fuzz/orphanage.cpp

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <net_processing.h>
+#include <primitives/transaction.h>
+#include <test/fuzz/FuzzedDataProvider.h>
+#include <test/fuzz/fuzz.h>
+#include <test/fuzz/util.h>
+#include <test/util/mining.h>
+#include <test/util/script.h>
+#include <test/util/setup_common.h>
+#include <txorphanage.h>
+#include <validation.h>
+
+namespace {
+const TestingSetup* g_setup;
+std::set<COutPoint> g_outpoints;
+} // namespace
+
+void initialize_orphanage()
+{
+    static const auto testing_setup = MakeNoLogFileContext<const TestingSetup>();
+    g_setup = testing_setup.get();
+    // initialize outpoints to construct transactions later
+    for (int i = 0; i < 4; ++i) {
+        CTxIn in = MineBlock(g_setup->m_node, P2WSH_OP_TRUE);
+        g_outpoints.insert(in.prevout);
```

Comment:
I think you can just create a few outpoints by calling the `COutPoint` constructor and passing a txid of 0 or 1 or so. This should be so cheap that it doesn't have to be done in the global setup, but can be done local in the fuzz target itself.